sdk/CoreWCF/Azure.CoreWCF.AzureQueueStorage/src/CoreWCF/Channels/AzureQueueStorageQueueNameConverter.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.Storage.CoreWCF.Channels
+{
+    /// <summary>
+    /// Class containing methods to convert Queue Name.
+    /// </summary>
+    public class AzureQueueStorageQueueNameConverter
+    {
+        /// <summary>
+        /// Method to fetch Queue Endpoint Url
+        /// </summary>
+        public static string GetEndpointUrl(string accountName, string queueName)
+        {
+            return $"net.aqs://{accountName}.queue.core.windows.net/{queueName}";
```

Comment:
I do not believe this works across Azure clouds and with Azure stack, as written.